### PR TITLE
Update documentation to include multiple-TLS support.

### PR DESCRIPTION
**Pull request by rramkumar1**
/assign @nicksardo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -381,7 +381,9 @@ The Ingress controller looks for a compatible readiness probe first, if it finds
 
 ## Frontend HTTPS
 
-For encrypted communication between the client to the load balancer, you need to specify a TLS private key and certificate to be used by the ingress controller.
+For encrypted communication between the client to the load balancer, you need to specify a TLS private key and certificate to be used by the ingress controller. 
+
+Version 1.1 of GLBC now supports (as a beta feature) using more than one SSL certificate in a single Ingress for request termination (aka Multiple-TLS). With this change, keep in mind that the upper limit is 10.
```
Comment (nicksardo) [inline README.md]: "GCP's limit is 10". Could you also link to the GCP page that says that, please

Comment (kimroen) [inline README.md]: How can a user of GKE know what version of the load balancer they are using to realise whether or not this information is relevant to them?

Comment (rramkumar1) [reply README.md]: Done.

Comment (rramkumar1) [reply README.md]: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/glbc.manifest

File: README.md [hunk 2]
```diff
@@ -431,6 +433,25 @@ spec:
...
 
+Specifying multiple secrets can be done as follows:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: no-rules-map
+spec:
+  tls:
```
Comment (stevedomin) [inline README.md]: How would you specify which certificate to use in an example with a rules map?

```
apiVersion: extensions/v1beta1
kind: Ingress
spec:
  tls:
  - secretName: svc1-certificate
  - secretName: svc2-certificate
  backend:
    serviceName: svc1
    servicePort: svc1-port
  rules:
  - host: svc1.example.com
    http:
      paths:
      - path: /*
        backend:
          serviceName: svc1
          servicePort: svc1-port
  - host: svc2.example.com
    http:
      paths:
      - path: /*
        backend:
          serviceName: svc2
          servicePort: svc2-port
```

Comment (rramkumar1) [reply README.md]: @stevedomin Should be updated now, PTAL!

Comment (stevedomin) [reply README.md]: Sorry @rramkumar1, my bad. Should have thought to look this up in the GCP docs.

Comment (rramkumar1) [reply README.md]: @stevedomin No worries! Does everything look good on your end? If so, we can probably check this in.

Comment (stevedomin) [reply README.md]: Looks great, got it working in my project. Thanks again!

**Pull request merged**